apps/design/frontend/src/ballot_screen.tsx

```diff
@@ -210,6 +210,10 @@ export function BallotScreen(): JSX.Element | null {
     ballotStyleId: BallotStyleId;
     precinctId: string;
   }>();
+  const { search } = useLocation();
+  const searchParams = new URLSearchParams(search);
+  const splitId = searchParams.get('splitId');
```

Comment:
Make specifying `splitId` possible in the `libs/hmpb` preview as well?

Reply:
The split config options in this PR can only be specified in VxDesign at the moment (not in an election definition as imported by `libs/hmpb/preview`). I'm not very familiar with the preview function but my sense is that supporting split config in `libs/hmpb/preview` would follow the pattern of editing the `main` script and be

* inflexible but easily understood
* flexible but complicated

I think the permanent solution should be to support import of split config options in the election definition, but I'm happy to add stopgap support after closing out right bubble position support!